test.js

```diff
@@ -18,3 +18,55 @@ test(function (t) {
 		t.assert(code === 0);
 	});
 });
+
+test('--json', function (t) {
+	var cp = childProcess.execFile('./cli.js', ['--json'], {
+		cwd: __dirname
+	});
```

Comment:
The point of using `execFile` was that you can just do all the below checks in the callback.